src/browser/screen.js

```diff
@@ -307,13 +325,20 @@ function ScreenAdapter(screen_container, bus)
         // Make sure to call this here, because pixels are transparent otherwise
         //screen.clear_screen();
 
-        graphic_image_data = graphic_context.createImageData(width, height);
+        graphic_image_data = graphic_context.createImageData(buffer_width, buffer_height);
         graphic_buffer = new Uint8Array(graphic_image_data.data.buffer);
         graphic_buffer32 = new Int32Array(graphic_image_data.data.buffer);
 
         graphical_mode_width = width;
         graphical_mode_height = height;
 
+        layers =
+        [{
+            screen_x: 0, screen_y: 0,
+            buffer_x: 0, buffer_y: 0,
+            buffer_width: width, buffer_height: height
+        }];
```

Comment:
This is the configuration for svga, right? In the reset state, shouldn't we start without layers?

Reply:
> This is the configuration for svga, right?

Yup, this is the configuration so that `update_buffer` behaves the same way as it used to.

> In the reset state, shouldn't we start without layers?

Hmm, good point. Starting without layers will help suppress drawing incorrect graphics when the video mode hasn't been fully configured yet, and it's more intuitive to draw nothing than to draw something potentially incorrect.